Detectors/MUON/MCH/Base/include/MCHBase/Digit.h

```diff
@@ -64,6 +67,7 @@ class Digit
 
  private:
   Time mTime;
+  uint16_t mNofSamples; /// number of samples in the signal
```

Comment:
Since members are added, I think this still needs a version increase in the ClassDef. I'll do it directly on the branch.

Reply:
Done, Version number is now 2.